plugins/by-name/neotest/adapters.nix

```
@@ -37,12 +37,10 @@ let
         mkIf cfg.enable {
           extraPlugins = [ cfg.package ];
 
-          assertions = [
-            {
-              assertion = config.plugins.neotest.enable;
-              message = "Nixvim: you have enabled `plugins.neotest.adapters.${name}` but `plugins.neotest.enable` is `false`.";
-            }
-          ];
+          assertions = lib.nixvim.mkAssertions "plugins.neotest.adapters.${name}" {
+            assertion = config.plugins.neotest.enable;
+            message = "You must set `plugins.neotest.enable` to `true` for using this extension.";
```

```suggestion
            message = "You must set `plugins.neotest.enable` to `true` to use this extension.";
```

However, maybe it would be good to standardize the verbiage for all of these `you have enabled x but not y` messages separately.